file-formats/sito/examples/TM_SITO_TRANS_TEST_01.sito.json

```diff
@@ -0,0 +1,410 @@
+        "language": "de"
+      },
+      {
+        "sitnobjectname": "TM_SITO_TRANS_TEST_01",
+        "sitnobjectdescription": "The booking object is used in the demo scenario Flight Profitability (SITDEMO_FLIGHT_PROFITABILITY).",
+        "language": "en"
+      }
+    ],
+    "sit2ObjStruc": [
+      {
+        "sitnobjstructype": "01",
+        "sitnobjstrucscope": "01",
+        "sitnobjstrucname": "I_SITN2DEMOBOOKING",
+        "sitnobjstrucid": "SITDEMO_CDS_BOOKING",
+        "sit2ObjStrucT": [
+          {
+            "sitnobjstrucdesc": "CDS view of booking (demo object)",
+            "language": "en"
+          }
+        ],
+        "sit2ObjStrSk": [
+          {
+            "sitnobjectfieldorder": "01",
+            "sitnobjectfield": "SITNDEMOCARRIERID"
+          },
+          {
+            "sitnobjectfieldorder": "02",
+            "sitnobjectfield": "SITNDEMOCONNECTIONID"
+          },
+          {
+            "sitnobjectfieldorder": "03",
+            "sitnobjectfield": "SITNDEMOFLIGHTDATE"
+          },
+          {
+            "sitnobjectfieldorder": "04",
+            "sitnobjectfield": "SITNDEMOBOOKINGID"
+          }
+        ],
+        "sit2OsVhMap": [
+          {
+            "sitnobjstrucvhapptype": "01",
+            "sitnobjstrucvhtype": "01",
+            "sitnobjstrucvhscope": "01",
+            "sitnobjstrucfieldname": "SITNDEMOCARRIERREGION"
+          },
+          {
+            "sitnobjstrucvhapptype": "01",
+            "sitnobjstrucvhtype": "01",
+            "sitnobjstrucvhscope": "01",
+            "sitnobjstrucfieldname": "SITNDEMOBOOKINGCLASS"
+          },
+          {
+            "sitnobjstrucvhapptype": "01",
+            "sitnobjstrucvhtype": "01",
+            "sitnobjstrucvhscope": "01",
+            "sitnobjstrucfieldname": "SITNDEMOCARRIERID"
+          },
+          {
+            "sitnobjstrucvhapptype": "01",
+            "sitnobjstrucvhtype": "01",
+            "sitnobjstrucvhscope": "01",
+            "sitnobjstrucfieldname": "SITNDEMOCONNECTIONID"
+          },
+          {
+            "sitnobjstrucvhapptype": "01",
+            "sitnobjstrucvhtype": "01",
+            "sitnobjstrucvhscope": "01",
+            "sitnobjstrucfieldname": "SITNDEMOAGENCYID"
+          },
+          {
+            "sitnobjstrucvhapptype": "01",
+            "sitnobjstrucvhtype": "01",
+            "sitnobjstrucvhscope": "01",
+            "sitnobjstrucfieldname": "SITNDEMOAGENCYREGION"
+          },
+          {
+            "sitnobjstrucvhapptype": "01",
+            "sitnobjstrucvhtype": "01",
+            "sitnobjstrucvhscope": "01",
+            "sitnobjstrucfieldname": "SITNDEMOBOOKINGID"
+          }
+        ],
+        "sit2OsVhMapS": [
+          {
+            "sitnobjstrucvhapptype": "01",
+            "sitnobjstrucfieldname": "SITNDEMOCARRIERREGION",
+            "sitnobjvhsrventityset": "C_Sitn2DemoFlightVH",
+            "sitnobjvhsrvproperty": "SitnDemoRegionCode",
```

Comment:
is the content of fields `sitnobjstrucfieldname` and`sitnobjvhsrvproperty` always the same(except the latter has camel-case values)? 

If so, would it make sense to only display the camel-case values?